card/mod/pointer/set/abstract/02_pointer/html_views.rb

```diff
@@ -5,6 +5,12 @@
     standard_pointer_core
   end
 
+  view :item_cores, cache: :never do
+    card.known_item_cards.map do |item|
+      nest item, view: :core
+    end.join "\n"
+  end
```

Comment:
seems like what we want is a view that is more like `bare` and then you specify the item views.  `nest :bare, items: {view: :core}`.  I don't like the idea of different views for different item views.